src/external_cli/rustup.rs

```diff
@@ -0,0 +1,47 @@
+//! Utilities for the `rustup` CLI tool.
+
+#![expect(dead_code, reason = "Will be used for build/run commands")]
+
+use std::process::Command;
+
+use dialoguer::Confirm;
```

Comment:
```suggestion
use std::process::Command;
use dialoguer::Confirm;
```

(The smallest, nittiest nit that ever knitted) I like to remove the space between these imports so that `rustfmt` will automatically sort them alphabetically. (Yay for opinionated code style!)

Reply:
I think this is intentional by rust-analyzer, it will separate for example `std` and external crates (but keep these groups sorted internally): https://rust-analyzer.github.io/manual.html#auto-import

Reply:
Good to know!